docs/GOVERNANCE_ACTION_SUBMISSION.md

```diff
@@ -12,6 +12,51 @@
 
 For creating the Governance Action, you need to consume 2 utility methods provided by `GovernanceActionProvided` (documented later within this document), and 3 exported from `CardanoProvider` wallet actions (2 for the 2 types of supported by GovTool Governance Actions and 1 for Signing and Submitting the transaction)
 
+### Types
+
+```typescript
+import { VotingProposalBuilder } from "@emurgo/cardano-serialization-lib-nodejs";
+
+interface GovernanceAction {
+  title: string;
+  abstract: string;
+  motivation: string;
+  rationale: string;
+  references: [{ title: string; url: string }];
```

Comment:
@MSzalowski Should it be `label` instead of the `title`, because in the cip-108 docs it is label and also in your test file?

Reply:
It should! Thank you :+1:

Reply:
```suggestion
  references: [{ label: string; url: string }];
```